checks/checks.md

```diff
@@ -16,6 +16,13 @@ A project which is not active may not be patched, may not have its dependencies
 **Remediation steps**
 - There is *NO* remediation work needed here. This is just to indicate your project activity and maintenance commitment.
 
+## Automatic-Dependency-Update 
+
+This check tries to determine if a project has dependencies automatically updated. The checks looks for [dependabot](https://dependabot.com/docs/config-file/) or [renovatebot](https://docs.renovatebot.com/configuration-options/).This check only looks if  it is enabled and does not ensure that it is run and PullRequests are merged. 
```

Comment:
nite spaces messed up in ".This check only looks if  it"
should be ". This check only looks if it"

Reply:
s/PullRequests/pull requests
no need to capitalize

Reply:
Thanks, took care of it.